### PR TITLE
#100 add message to inform user when the import calendar is finishing

### PR DESCRIPTION
**Pull request by asmaNode**
bugfix2

### CODE REVIEW

File: src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ function CalCalendarImportController(
   function submit() {
     var notificationMessages = {
       progressing: 'Submitting importing calendar request...',
-      success: 'Request submitted',
+      success: 'An email will be sent when the import is finishing',
```
Comment (rezk2ll) [inline src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js:62]:
```suggestion
      success: 'You will receive an email when the import is finished',
```

Comment (asmaNode) [reply src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js:62]: Based on other ux example , will be better to you this format ' an email will .... '

**Review by rezk2ll — APPROVED** [review]

**Review by renaudboyer — DISMISSED** [review]

File: src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js (hunk 1 above)
Comment (rezk2ll) [reply src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js]: Yes, an email will be sent to who? It is not specified in the string.

File: src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ function CalCalendarImportController(
   function submit() {
     var notificationMessages = {
       progressing: 'Submitting importing calendar request...',
-      success: 'Request submitted',
+      success: 'An email will be sent when the import is finished',
```
Comment (tuanlc) [inline src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js:62]: Good improvement but you should add translations for it

```suggestion
      success: 'Calendar is importing, you will receive a reporting email when the import is finished',
```

**Review by tuanlc — CHANGES_REQUESTED** [review]

File: src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js (hunk 2 above)
Comment (huy-ta) [reply src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js:62]: @asmaNode You should make the message clearer:

> The calendar is being imported. You will receive a report email when the import is finished.

I'll give you the Vietnamese translation also:

> Lịch đang được nhập. Bạn sẽ nhận được một thư báo cáo khi quá trình nhập hoàn thành.

File: src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js [hunk 3]
```diff
@@ -58,9 +59,9 @@ function CalCalendarImportController(
 
   function submit() {
     var notificationMessages = {
-      progressing: 'Submitting importing calendar request...',
-      success: 'Request submitted',
-      failure: 'Failed to submit request'
+      progressing: esnI18nService.translate('Submitting importing calendar request'),
```
Comment (huy-ta) [inline src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js:62]:
```suggestion
      progressing: esnI18nService.translate('Submitting importing calendar request...'),
```

By the way I'm not sure if you don't need to `esnI18nService.translate('Submitting importing calendar request...').toString()` here.

Comment (asmaNode) [reply src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js:62]: toString() why need to parse as string , what is the difference whith other texts ?

**Review by rezk2ll — APPROVED** [review]

File: src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js (hunk 3 above)
Comment (tuanlc) [reply src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js:62]: @asmaNode the `asyncAction` service translates texts by itself: https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/blob/main/src/frontend/js/modules/async-action.js#L49

Then, we do not need to translate text before calling it.

**Review by rezk2ll — APPROVED** [review]

File: src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js [hunk 4]
```diff
@@ -58,8 +58,8 @@ function CalCalendarImportController(
 
   function submit() {
     var notificationMessages = {
-      progressing: 'Submitting importing calendar request...',
-      success: 'Request submitted',
+      progressing: 'Submitting importing calendar request',
```
Comment (huy-ta) [inline src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js:61]: It's a convention that we add "..." to the end of the message if it's a message about something that's being done (for example "Loading...", "Submitting...").
So I think you should keep the "..." dots.

File: src/linagora.esn.calendar/i18n/en.json [hunk 5]
```diff
@@ -229,13 +230,15 @@
   "Some attendees are not available": "Some attendees are not available",
   "Some meeting information contained in this message cannot be processed.": "Some meeting information contained in this message cannot be processed.",
   "Subscription": "Subscription",
+  "Submitting importing calendar request":"Submitting importing calendar request",
```
Comment (huy-ta) [inline src/linagora.esn.calendar/i18n/en.json:233]:
```suggestion
  "Submitting importing calendar request": "Submitting importing calendar request",
```

File: src/linagora.esn.calendar/i18n/en.json [hunk 6]
```diff
@@ -229,13 +230,15 @@
...
   "Successfully subscribed to calendar": "Successfully subscribed to calendar",
   "Successfully subscribed to calendars": "Successfully subscribed to calendars",
   "Tentative": "Tentative",
   "Tentatively accepted: %s (%s)": "Tentatively accepted: %s (%s)",
   "The reservation of resource {{name}} has been accepted": "The reservation of resource {{name}} has been accepted",
   "The reservation of resource {{name}} has been declined": "The reservation of resource {{name}} has been declined",
   "The resource reservation event can not be retrieved.": "The resource reservation event can not be retrieved.",
+  "The calendar is being imported. You will receive a report email when the import is finished.":"The calendar is being imported. You will receive a report email when the import is finished.",
```
Comment (huy-ta) [inline src/linagora.esn.calendar/i18n/en.json:241]:
```suggestion
  "The calendar is being imported. You will receive a report email when the import is finished.": "The calendar is being imported. You will receive a report email when the import is finished.",
```

**Review by huy-ta — APPROVED** [review]

**Review by tuanlc — APPROVED** [review]

**Review by rezk2ll — APPROVED** [review]

**Pull request merged**